openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -364,9 +365,18 @@ def sync_project_from_kitsu(dbcon: AvalonMongoDB, project: dict):
         if naming_pattern.match(item["name"])
     ]
 
+    # Get all statuses for projects from Kitsu
+    all_status = gazu.project.all_project_status()
+    for status in all_status:
+        if project['project_status_id'] == status['id']:
+            project['project_status_name'] = status['name']
+
     # Sync project. Create if doesn't exist
     bulk_writes.append(write_project_to_op(project, dbcon))
 
+    if project['project_status_name'] == "Closed":
+        return
+
```

Comment:
With this PR we're now also writing **closed projects** into OpenPype database that have never been synced before. I wonder whether we should only sync a closed project _IF_ it exists in OpenPype **and** is currently active in OpenPype.

I'd then also move all the asset+shot+episode etc. querying to AFTER knowing whether we want to sync the project. That way we'd only sync to deactivate a project in OP once a project is closed, and otherwise SYNC everything again as soon as the project is active again.

This would make it much lighter, keep it faster as well as soon as you have e.g. 15 closed projects.